aws/k8s-aws-alb-script.sh

```diff
@@ -71,25 +71,41 @@ sleep 10
 
 EKS_CLUSTER_VERSION=$(aws eks describe-cluster --name $CLUSTERNAME --region $AWS_REGION --query cluster.version --output text)
 
-# echo "apply -f k8s/secret-challenge-vault-service.yml in 10 s"
-# sleep 10
-# kubectl apply -f k8s/secret-challenge-vault-service.yml
-echo "apply -f k8s/wrongsecrets-balancer-service.yml in 10 s"
+EXTERNAL_DNS_ROLE_ARN="$(terraform output -raw external_dns_role_arn)"
+kubectl create serviceaccount -n kube-system external-dns
+kubectl annotate serviceaccount -n kube-system --overwrite external-dns eks.amazonaws.com/role-arn=${EXTERNAL_DNS_ROLE_ARN}
+
+echo "apply -f k8s/external-dns-*.yaml in 10 s"
```

Comment:
can we make this optional under a flag, so that the default would be "you get an alb assigned dns entry"?